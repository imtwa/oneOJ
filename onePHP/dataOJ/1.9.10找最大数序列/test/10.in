28
17,25,14,26,11,24,17,19,22,23,26,21,13,25
13,12,14,10,11,13,22,24,16,17,23,11,10,12
18,25,10,29,10,14,21,10,19,11,28,25,19,10
18,27,19,29,29,12,16,24,18,23,19,17,24,18
15,19,12,16,11,10,12,11,30,25,20,12,24,19
24,12,29,18,15,17,28,30,17,12,29,10,18,17
16,13,30,28,28,30,17,11,13,30,21,14,10,11
30,22,21,21,13,16,27,13,22,10,15,12,18,24
15,15,12,11,17,27,15,21,27,24,13,23,28,29
14,20,19,29,29,20,19,10,25,24,16,21,19,25
14,25,25,28,21,30,27,21,27,29,27,20,28,23
12,16,30,11,21,27,18,10,18,30,11,27,26,23
27,15,15,28,11,21,25,20,29,10,22,29,21,28
22,22,25,12,11,19,12,28,18,24,14,24,18,10
16,12,16,26,15,18,20,20,28,12,21,22,17,25
13,13,16,18,12,22,18,17,18,17,29,17,25,11
26,22,28,18,14,19,12,17,27,26,19,21,29,16
29,16,21,28,10,15,23,12,18,27,23,14,10,15
10,26,27,10,27,13,10,26,24,18,29,27,29,29
11,17,15,27,20,18,10,23,14,29,18,11,25,28
11,21,19,21,20,22,21,12,25,21,30,18,18,13
11,14,24,26,10,14,13,28,25,20,20,24,13,11
28,25,11,22,21,20,15,18,29,17,24,11,18,27
15,20,15,23,15,25,22,28,30,27,27,25,22,18
19,13,14,28,19,26,18,30,19,16,19,12,13,20
13,11,21,22,13,11,28,29,28,16,10,14,20,22
27,21,24,23,29,15,13,17,10,24,15,25,22,18
10,22,28,18,28,14,10,17,30,16,17,14,25,19
