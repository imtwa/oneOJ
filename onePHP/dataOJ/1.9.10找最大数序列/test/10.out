30
5,6,7,8,11,12,21,24,25,28